### ISSUE TITLE
Fix bootstrap nodes open several connections

### ISSUE THREAD

**Issue by lrubiorod** [witnet/witnet-rust#1220]
We have noticed that bootstrap nodes are creating more than one connection and it produces that the inbound list of the other nodes are full with connections copies of the bootstrap nodes.

When the bootstrap nodes "restart" a connection with an outbound, looks like the connection continues opened, so the outbound now has one more connection, the old one that is not closed and the new one. That process occurs always that a bootstrap restart for any reason (manual, panic...)

**Comment by aesedepece** [comment]
Side note: we couldn't confirm if this is indeed an error in `witnet-rust`. It could well be something specific to our Docker / Portainer / Azure setup.

**Comment by drcpu-github** [comment]
Writing a quick comment on this as it seems I am experiencing this error (still / again?) with my node that is set as a bootstrap node in witnet.toml config file.

I had this happen with 100% reproducability before the testnet launched. It happened every time I started my node before the network bootstrapped in the following order:
1) I had around 40 / 50 incoming connections
2) After some time I got the full 12 outbound connections necessary to synchronize
3) Between 1 and 5 minutes after that, the node starts printing out that it has too many open files
`[ERROR witnet_node::actors::connections_manager] Error incoming listener: Too many open files (os error 24)`
It never happened before my node had 12 outbound connections.

My node crashed a couple of minutes ago and got restarted automatically after which it again experienced this error. Restarting it manually did solve it. Hence it seems it also happens after network bootstrap.

I did also try increasing the open file limit to 16k, which should be more than enough (or so I thought given the size of the network). Maybe I should just set it to a 1M.
